credentials/src/main/kotlin/web5/sdk/credentials/model/Validator.kt

```diff
@@ -0,0 +1,125 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.nfeld.jsonpathkt.JsonPath
+import com.nfeld.jsonpathkt.extension.read
+import web5.sdk.credentials.JsonPathParseException
+
+/**
+ * PresentationDefinitionV2Validator Validator.
+ **/
+public object PresentationDefinitionV2Validator {
+
+  /**
+   * Validates a PresentationDefinitionV2.
+   */
+  public fun validate(presentationDefinition: PresentationDefinitionV2) {
+    require(presentationDefinition.id.isNotEmpty()) { "PresentationDefinition id must not be empty" }
+    presentationDefinition.name?.let { require(it.isNotEmpty()) {
+      "PresentationDefinition name must not be empty" }
```

Comment:
The approach we've settled on is to use typed exceptions for this sort of stuff. See https://github.com/TBD54566975/web5-kt/blob/48ce6764163b32714be7d8282bfad0678f49debb/credentials/src/main/kotlin/web5/sdk/credentials/ExceptionDeclarations.kt#L10 for an example definition and use. 

Would it make sense to move towards having definitions for all exceptions thrown?

Reply:
Yess will update 👍